src/helpers/helpers.dom.ts

```diff
@@ -229,24 +304,28 @@ export function retinaScale(chart, forceRatio, forceStyle) {
  * https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Safely_detecting_option_support
  * @private
  */
+// eslint-disable-next-line wrap-iife
 export const supportsEventListenerOptions = (function() {
   let passiveSupported = false;
   try {
     const options = {
-      get passive() { // This function will be called when the browser attempts to access the passive property.
+      get passive() {
+        // This function will be called when the browser attempts to access the passive property.
         passiveSupported = true;
         return false;
-      }
+      },
     };
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
     // @ts-ignore
     window.addEventListener('test', null, options);
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
     // @ts-ignore
     window.removeEventListener('test', null, options);
   } catch (e) {
     // continue regardless of error
   }
   return passiveSupported;
-}());
```

Comment:
Why was this change made?

Reply:
this (and other indentation stuff) was a prettier run accidentally, just fixed it